src/containers/Identity/PersonalInfo/ClaimManager/View.js

```diff
@@ -0,0 +1,89 @@
+import React, { useCallback } from 'react';
+import { Map as IMap } from 'immutable';
+import { View, Text, Platform } from 'react-native';
+import { ScrollView, TouchableOpacity } from 'react-native-gesture-handler';
+import Icon from 'react-native-vector-icons/Ionicons';
+import MaterialIcons from 'react-native-vector-icons/MaterialIcons';
+import { CheckBox } from 'react-native-elements';
+import styles from './styles';
+import { truncateString } from './utils/truncateString';
+
+const ClaimManager = (props) => {
+  const {
+    claims,
+    selectedClaims,
+    navigation,
+    claimCategories,
+    actions: {
+      setClaimVisibility,
+      updateSelectedClaims,
+      clearSelectedClaims,
+    },
+  } = props;
+
+  const toggleClaimVisibility = (selectedClaim) => {
+    setClaimVisibility(selectedClaim.get('id', ''), !selectedClaim.get('hidden'));
+  };
+
+  const moveSingleClaim = (selectedClaim) => {
+    clearSelectedClaims();
+    updateSelectedClaims(selectedClaim);
+    navigation.navigate('MoveIntoCategory');
+  };
+
+  const moveSelectedCategories = () => {
+    navigation.navigate('MoveIntoCategory');
+  };
+
+  const selectClaim = (selectedClaim) => {
+    updateSelectedClaims(selectedClaim);
+  };
+
+  const checkIfClaimIsSelected = useCallback((claim) => selectedClaims.includes(claim), [selectedClaims]);
+
+  const getCategotyName = (name) => {
```

Comment:
This is an util function, move it outside of the component.

Reply:
done